Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "Oceananigans"
 uuid = "9e8cae18-63c1-5223-a75c-80ca9d6e9a09"
 authors = ["Climate Modeling Alliance and contributors"]
-version = "0.87.2"
+version = "0.88.0"
```

Comment:
What are the breaking changes? This PR seems to concern only MPI which doesn't really have user facing stuff?

Reply:
I changes a lot of the internals of the code so I think it is good to bump the version. There are some API changes such as:
- `active_cells_map` kwarg for `ImmersedBoundaryGrid`
- `boundary_buffer` for scalar diffusivities
and one breaking change
- size passed to distributed grids is the _local_ not the _global_ size